packages/side-nav/README.md

```diff
@@ -155,10 +155,14 @@ return (
 
 ## Properties
 
-| Prop        | Type     | Description                                                  | Default     |
-| ----------- | -------- | ------------------------------------------------------------ | ----------- |
-| `className` | `string` | Class name that will be applied to the root-level element.   | `undefined` |
-| `children`  | `node`   | Content that will be rendered inside the root-level element. | `undefined` |
+| Prop            | Type                                            | Description                                                                     | Default     |
+| --------------- | ----------------------------------------------- | ------------------------------------------------------------------------------- | ----------- |
+| `className`     | `string`                                        | Class name that will be applied to the root-level element.                      | `undefined` |
+| `children`      | `node`                                          | Content that will be rendered inside the root-level element.                    | `undefined` |
+| `baseFontSize`  | `14`, `16`                                      | Determines the base font size (in pixels) of the Side Nav                       | `14`        |
+| `widthOverride` | `number`                                        | Width (in pixels) of Side Navigation.                                           | `184`       |
```

Comment:
We should call out these docs-specific props as not being for general use. The last thing we want is for one of the cloud products to override the width.

Also, should we allow this to be set to a percentage? E.g. if they pass a string, we expect the unit to be set (`px`, `%`, `vw`, etc), and if they pass a number, it's interpreted as pixels.

Reply:
Should I leave them out of the docs? If not, I can write *Docs only*? What do you think?